docs/observers/docker.md

```diff
@@ -21,7 +21,8 @@ Observer Type: `docker`
 | Config option | Required | Type | Description |
 | --- | --- | --- | --- |
 | `dockerURL` | no | `string` |  (**default:** `unix:///var/run/docker.sock`) |
-| `pollIntervalSeconds` | no | `integer` | How often to poll the docker API (**default:** `10`) |
```

Comment:
Should it be documented that the `pollIntervalSeconds` option is no longer supported, in case there are any customers that are using this and then upgrades?

Reply:
Yes, that will go in the release notes as a Breaking Change.